app/renderer/actions/Inspector.js

```diff
@@ -500,3 +517,103 @@ export function setActionArg (index, value) {
     dispatch({type: SET_ACTION_ARG, index, value});
   };
 }
+
+/**
+ * Ping server every 30 seconds to prevent `newCommandTimeout` from killing session
+ */
+export function runKeepAliveLoop () {
+  return (dispatch, getState) => {
+    dispatch({type: SET_LAST_ACTIVE_MOMENT, lastActiveMoment: +(new Date())});
+    const {driver} = getState().inspector;
+
+    const keepAliveInterval = setInterval(async () => {
+      const {lastActiveMoment} = getState().inspector;
+      console.log('Pinging Appium server to keep session active'); // eslint-disable-line no-console
+      try {
+        await driver.getTimeouts(); // Pings the Appium server to keep it alive
+      } catch (ign) {}
+      const now = +(new Date());
```

Comment:
why not to use Date.now like below?

Reply:
because I was just moving the existing code! but i can update to keep it consistent.